certbot/CHANGELOG.md

```diff
@@ -68,7 +68,8 @@ More details about these changes can be found on our GitHub repo.
 
 ### Fixed
 
-*
+* Revoking a certificate based on an ECDSA key can now be done with `--key-path`.
+  See [GH #8569](https://github.com/certbot/certbot/issues/8569).
```

Comment:
This should be moved to the current master section of the changelog instead of the section for our 1.21.0 release in November. Other than that, this PR LGTM! Thanks for picking this up.

When you fix this, [please do not force push](https://eff-certbot.readthedocs.io/en/stable/contributing.html#submitting-a-pull-request) and instead just add a new commit. I can clean things up when I squash & merge the PR.